src/bearhug.js

```diff
@@ -174,7 +173,7 @@ angular.module('talis.bearhug', [])
     /**
      * Routes/codes for which to disable the global error handler that shows the red error page.
      */
-    $httpProvider.responseInterceptors.push([
+    $httpProvider.interceptors.push([
```

Comment:
My understanding of `http.interceptors` is that it needs an object specifying the part of the HTTP cycle you wish to intercept (`request`, `requestError`, `response`, `responseError`).  I believe you need to [provide an object to override only responses](https://docs.angularjs.org/api/ng/service/$http#interceptors), and I'm not even sure whether using a bare function will work.  Does it?